.github/dependabot.yml

```diff
@@ -16,6 +16,11 @@ updates:
       - "jglick"
     schedule:
       interval: "daily"
+    ignore:
+      # TODO https://github.com/jenkinsci/bom/pull/1374#issuecomment-1205302760 pending new version
+      - dependency-name: trilead-api
```

Comment:
https://github.com/jenkinsci/bom/pull/1382#issuecomment-1207121920

Reply:
a86fa42

Reply:
https://github.com/jenkinsci/trilead-api-plugin/releases/tag/2.72.v2a_3236754f73 includes https://github.com/jenkinsci/trilead-api-plugin/pull/55 which I guess is now safe for newer lines @kuisathaverat?

Reply:
yes, it is safe for 2.261+